src/lib.rs

```diff
@@ -100,6 +101,78 @@ pub fn build<P: AsRef<Path>>(path: P) -> PathBuf {
 }
 
 impl Config {
+    /// Return explicitly set profile or infer `CMAKE_BUILD_TYPE` from Rust's compilation profile.
+    ///
+    /// * if `opt-level=0` then `CMAKE_BUILD_TYPE=Debug`,
+    /// * if `opt-level={1,2,3}` and:
+    ///   * `debug=false` then `CMAKE_BUILD_TYPE=Release`
+    ///   * otherwise `CMAKE_BUILD_TYPE=RelWithDebInfo`
+    /// * if `opt-level={s,z}` then `CMAKE_BUILD_TYPE=MinSizeRel`
+    pub fn get_profile(&self) -> Cow<str> {
```

Comment:
I think this could return `&str` perhaps? I don't think that there's anywhere that necessarily needs to return a `String`

Reply:
You're right, of course. Done.